Jenkinsfile

```diff
@@ -72,89 +89,30 @@ try {
 
             // Lock required around all usages of Gradle as it isn't
             // able to share its cache between builds.
-            lock("${env.NODE_NAME}-android") {
-
-              stage('JVM tests') {
-                try {
-                  withCredentials([[$class: 'FileBinding', credentialsId: 'c0cc8f9e-c3f1-4e22-b22f-6568392e26ae', variable: 'S3CFG']]) {
-                    sh "chmod +x gradlew && ./gradlew assemble check javadoc -Ps3cfg=${env.S3CFG} ${abiFilter} --stacktrace"
-                  }
-                } finally {
-                  storeJunitResults 'realm/realm-annotations-processor/build/test-results/test/TEST-*.xml'
-                  storeJunitResults 'examples/unitTestExample/build/test-results/**/TEST-*.xml'
-                  step([$class: 'LintPublisher'])
-                }
+            if (useEmulator) {
+              // TODO: We should wait until the emulator is online. For now assume it starts fast enough
+              // before the tests will run.
+              sh '''yes "\n" | avdmanager create avd -n CIEmulator -k "system-images;android-29;default;x86" --force'''
```

Comment:
why no 64-bit image for the emulator?

Reply:
I'm running into all sorts of problems with the 64-bit emulator. I'm running into this error described here: https://github.com/mingchen/docker-android-build-box/issues/18

So far, it seems the only solution is running arm emulators instead on Docker. I _really_ hope we don't need to do that because I suspect that would be unbearable slow.

From what I can tell, the only other solution is havin dedicated machines that run emulators, because then they can run as part of the host.